metadata-ingestion/src/datahub/ingestion/source/salesforce.py

```diff
@@ -124,6 +124,9 @@ class SalesforceConfig(DatasetSourceConfigMixin):
         default=dict(),
         description='Regex patterns for tables/schemas to describe domain_key domain key (domain_key can be any string like "sales".) There can be multiple domain keys specified.',
     )
+    api_version: Optional[str] = Field(
```

Comment:
Can you set a default value?

Reply:
We are not setting default value on purpose, script logic will pick the latest api version in such case.